hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/authority/CertificateApprover.java

```diff
@@ -37,20 +38,9 @@ public interface CertificateApprover {
    * Approves a Certificate Request based on the policies of this approver.
    *
    * @param csr - Certificate Signing Request.
-   * @return - Future that will be contain the certificate or exception.
+   * @return - Future that will contain the certificate or exception.
```

Comment:
Future that will contain Void if request is valid, otherwise exception.

Reply:
Thank you for spotting this, I am updating the API doc shortly.